src/App.js

```diff
@@ -1,10 +1,22 @@
 import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import Home from './components/Home/Home';
+import BookAppointment from './components/BookAppointment/BookAppointment';
+import MyReservations from './components/MyReservations/MyReservations';
+import DoctorDetails from './components/DoctorDetails/DoctorDetails';
+
+// import Navigation from './components/Navigation/Navigation';
 
 function App() {
   return (
-    <div className="App">
-      Home
-    </div>
+    <>
+      <Routes>
+        <Route path="/" exact element={<Home />} />
+        <Route path="/BookAppointment" exact element={<BookAppointment />} />
+        <Route path="/MyReservations" exact element={<MyReservations />} />
+        <Route path="/DoctorDetails" exact element={<DoctorDetails />} />
```

Comment:
Let's keep all route names in small letters to follow best practices e.g `/book-appointment`